src/app/github-link/github-link.component.html

```diff
@@ -0,0 +1,9 @@
+<span>
```

Comment:
what is this extra `<span>` for? You should not need to wrap the `<a>` in a `<span>`.